content/en/docs/Installation/automated.md

```diff
@@ -142,6 +142,11 @@ $ docker run -v $PWD:/cOS -v /var/run/docker.sock:/var/run/docker.sock -i --rm q
 {{% /tab %}}
 {{< /tabpane >}}
 
+{{% alert title="Cloud config" %}}
+In the case of Auroraboot, make sure that the cloud config that you are mounting in the container (`-v $PWD/cloud_init.yaml:/cloud_init.yaml`) exists. Otherwise docker will create en empty directory to mount it on the container without any warnings and you will end up with an empty cloud config.
```

Comment:
```suggestion
In the case of Auroraboot, make sure that the cloud config that you are mounting in the container (`-v $PWD/cloud_init.yaml:/cloud_init.yaml`) exists. Otherwise docker will create an empty directory to mount it on the container without any warnings and you will end up with an empty cloud config.
```
found another one, sorry :)